data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -52,7 +53,8 @@
 
     private static final String SEPARATOR = "/";
 
-    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final ObjectMapper mapper = new ObjectMapper()
```

Comment:
This shouldn't be necessary when we use Java 8 time.

Reply:
I think we should register `com.fasterxml.jackson.datatype.jsr310.JavaTimeModule` if we use Java 8 time.

Reply:
If they do not apply that module automatically, then yes.